examples/flows/single-transfer/input-schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+        "source_endpoint_id",
+        "destination_endpoint_id",
+        "transfer_items"
+    ],
+    "properties": {
+        "source_endpoint_id": {
+            "type": "string"
+        },
+        "destination_endpoint_id": {
+            "type": "string"
+        },
+        "transfer_items": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "source_path": {
+                        "type": "string"
+                    },
+                    "destination_path": {
+                        "type": "string"
+                    }
+                }
```

Comment:
Is it possible to specify that these properties are required? Are other properties allowed?

Reply:
Yes and yes! I updated the schema to match the transfer_items schema definition on the transfer/transfer Action Provider. It's now exactly what's defined over there.